### DISCUSSION TITLE
Server Error : 'str' object has no attribute 'get'

### DISCUSSION THREAD

**Discussion by kgw0820** [netbox-community/netbox#14388] [Help Wanted!]
This error occurred during Default Dashboard modification. 
I have restored it to its original state, but the error persists.

![image](https://github.com/netbox-community/netbox/assets/141695004/df17d5df-3986-447a-b611-ac07cb28bee2)

What steps should I take?

Help me, please.

**Comment by candlerb** [comment]
What page are you looking at when you see this error, i.e. what's in the URL bar of your browser?

Can you get a full backtrace? You can either set ADMINS and EMAIL in configuration.py (in which case it'll be mailed out to you), or temporarily set DEBUG=True and you'll get it in the browser.
